### ISSUE TITLE
ArgumentException: An item with the same key has already been added. Key: RecaptchaSettings (Parameter 'propertyName')

### ISSUE THREAD

**Issue by jagbarcelo** [griesoft/orchardcore-recaptcha#10]
We are trying to upgrade from OrchardCore 1.8.4 + Griesoft.OrchardCore.ReCaptcha 1.0.0 to the newest versions for both of them (OrchardCore 2.0.2 + Recaptcha 2.0.0).

We have tried in two different ways:

a) Straightforward upgrade for both and,
b) Disable ReCaptcha component in OrchardCore before upgrading both packages (and then try to enable it back again)

In both cases we get the following error when OC starts (homepage):
```
System.ArgumentException: An item with the same key has already been added. Key: RecaptchaSettings (Parameter 'propertyName')
   at System.Text.Json.ThrowHelper.ThrowArgumentException_DuplicateKey(String paramName, String propertyName)
   at System.Text.Json.Nodes.JsonObject.InitializeDictionary()
   at System.Text.Json.Nodes.JsonObject.TryGetPropertyValue(String propertyName, JsonNode& jsonNode)
   at OrchardCore.Entities.EntityExtensions.As[T](IEntity entity, String name)
   at OrchardCore.Settings.SiteSettings.As[T]()
   at OrchardCore.Settings.SiteServiceExtensions.GetSettingsAsync[T](ISiteService siteService)
   at OrchardCore.Media.Processing.MediaTokenSettingsUpdater.ActivatedAsync()
   at OrchardCore.Environment.Shell.Scope.ShellScope.<>c.<<ActivateShellInternalAsync>b__43_0>d.MoveNext()
--- End of stack trace from previous location ---
   at OrchardCore.Environment.Shell.Scope.ShellScope.UsingAsync(Func`2 execute, Boolean activateShell)
   at OrchardCore.Environment.Shell.Scope.ShellScope.UsingAsync(Func`2 execute, Boolean activateShell)
   at OrchardCore.Environment.Shell.Scope.ShellScope.UsingAsync(Func`2 execute, Boolean activateShell)
   at OrchardCore.Environment.Shell.Scope.ShellScope.UsingAsync(Func`2 execute, Boolean activateShell)
   at OrchardCore.Environment.Shell.Scope.ShellScope.UsingAsync(Func`2 execute, Boolean activateShell)
   at OrchardCore.Environment.Shell.Scope.ShellScope.ActivateShellInternalAsync()
   at OrchardCore.Environment.Shell.Scope.ShellScope.ActivateShellInternalAsync()
   at OrchardCore.Environment.Shell.Scope.ShellScope.UsingAsync(Func`2 execute, Boolean activateShell)
   at OrchardCore.Environment.Shell.Scope.ShellScope.UsingAsync(Func`2 execute, Boolean activateShell)
   at OrchardCore.Environment.Shell.Scope.ShellScope.UsingAsync(Func`2 execute, Boolean activateShell)
   at OrchardCore.Environment.Shell.Scope.ShellScope.UsingAsync(Func`2 execute, Boolean activateShell)
   at OrchardCore.Environment.Shell.Scope.ShellScope.UsingAsync(Func`2 execute, Boolean activateShell)
   at OrchardCore.Modules.ModularTenantContainerMiddleware.Invoke(HttpContext httpContext)
   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddlewareImpl.Invoke(HttpContext context)
```

Please, note that in the b) option, the second part is between parenthesis. That's because we are not able to do that second action.

OrchardCore simply does not start and the exception page shown above is what we see.

Are there any documented steps for this upgrade? Thanks.

**Comment by jagbarcelo** [comment]
Furthermore, even commenting the whole PackageReference in my project, the Exception persists:

<img width="1325" alt="image" src="https://github.com/user-attachments/assets/71664115-f269-4d04-984b-e71594a71cc1">

Can you find any reason for that behaviour? Is this package already built-in in the newest version of OrchardCore?

<img width="884" alt="image" src="https://github.com/user-attachments/assets/aaae8649-a89e-458b-9545-e4e3c6426193">

Thanks.

**Comment by jagbarcelo** [comment]
Nevermind.

By using https://sqlitebrowser.org/, I edited the `yessql.db` file directly and removed the `RecaptchaSettings` that the exception was complaining about.

For those that might have the same isssue, just edit the table `Document`, and find the one with `Type = OrchardCore.Settings.SiteSettings, OrchardCore.Settings`. Then edit the `Content` for that record (it is a simple big json text) and remove the `RecaptchaSettings`.

After that, OrchardCore just run fine (I had to set up again Griesoft.OrchardCore.ReCaptcha).
